cmd/acb/build.go

```diff
@@ -82,9 +82,12 @@ func newBuildCmd(out io.Writer) *cobra.Command {
 	f.StringArrayVar(&r.secretBuildArgs, "secret-build-arg", []string{}, "set secret build arguments")
 	f.StringArrayVar(&r.labels, "label", []string{}, "set metadata for an image")
 
+	// Runner never passed `username` and `password` now. Instead it adds it to `credentials` IFF passed.
```

Comment:
acb doesn't depend on runner. Maybe the comment can just mention the two parameters are deprecated.